docs/how-tos/how_to_modify_valid_exchanges_in_returns.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 1
+---
+
+# How to modify valid exchanges in returns
```

Comment:
I think "valid exchange items" would work better to avoid any possible confusion here.

Reply:
I agree it's better. Done.